source/mainnet/smart-contracts/tutorials/sponsoredTransactions/sponsoredTransactions_dapp.rst

```diff
@@ -0,0 +1,165 @@
+   can not be reused a second time.
+
+Submit a sponsored ``updateOperator`` transaction
+=================================================
+
+.. image:: ./images/updateOperator.gif
+   :alt: StreamPlayer
+   :align: center
+
+When exploring the front-end code, you will realize that the input parameter to the ``permit`` function is rather complex as seen below:
+
+.. code-block::
+
+   const operatorAction = addOperator
+      ? {
+         Add: [],
+      }
+      : {
+         Remove: [],
+      };
+
+   const message = {
+      contract_address: {
+         index: Number(SPONSORED_TX_CONTRACT_INDEX),
+         subindex: 0,
+      },
+      entry_point: 'updateOperator',
+      nonce: Number(nonce),
+      payload: {
+         UpdateOperator: [
+            [
+               {
+                  operator: {
+                     Account: [operator],
+                     },
+                     update: operatorAction,
+                  },
+               ],
+            ],
+         },
+      timestamp: EXPIRY_TIME_SIGNATURE,
+   };
+
+You can use the format returned by the ``concordium-client`` with the below command to have a
+guideline on how to create the above object:
+
+.. code-block:: console
+
+   $concordium-client contract show 4184 --grpc-port 10000 --grpc-ip node.testnet.concordium.com
+
+Submit a sponsored ``transfer`` transaction
+===========================================
+
+.. image:: ./images/transfer.gif
+   :alt: StreamPlayer
+   :align: center
+
+.. note::
+
+   The NFT smart contract used in the hosted demo app is meant for exploring sponsored transactions.
+   To ease this process, the mint function can be called by any account to mint a token.
+   ``Token_ids`` are stored in the smart contract in little-endian order and a counter is used to
+   increase the  ``token_id`` by 1 every time the mint function is invoked. The wallet displays the ``token_id`` bytes in a hex string.
+   In other words, the first token minted will have the ``token_id=01000000``,
+   the second token minted will have ``02000000``, ..., ``0e000000``, ``10000000``, ... .
+   You can lock up your minted ``token_id`` in the |bw| by getting the smart contract
+   index as seen below and searching for tokens in that contract that are owned by your account.
+
+.. image:: ./images/SmartContractIndex.png
+   :width: 40 %
+
+.. image:: ./images/ManageTokenTab.png
+   :width: 40 %
+
+.. note::
+
+   Every ``token_id`` would usually have
+   its own metadata associated with it by e.g. appending the ``TOKEN_METADATA_BASE_URL`` with the ``token_id``. But to simplify the
+   metadata management for Concordium, the NFT smart contract used in the hosted demo app returns
+   the same metadata hash (``TOKEN_METADATA_BASE_URL``) for all ``token_ids`` (no appending of ``token_ids``).
+   The NFT picture in the below URL changes every few seconds which is why you will see the |bw| updating the picture as well.
```

Comment:
The picture changes every time you access `https://picsum.photos/40/40` I think.  You get a random one each time.